src/components/ProjectListItem/index.jsx

```diff
@@ -0,0 +1,91 @@
+import cls from 'classnames';
+import { useState, useEffect } from 'react';
+import ProjectStatusIcon from '../ProjectStatusIcon';
+import styles from './styles.module.css';
+import store from '../../store';
+import {
+  borders,
+  effects,
+  flex,
+  layout,
+  padding,
+  margin,
+  colors,
+  typography as type,
+} from '../../theme';
+
+export default function ProjectListItem({
```

Comment:
Again, you'll se here that `projectDetail` doesn't really exist for an archive project list item, so I kinda fake it by only using the data you need for an archived project, which is it's slug from S3 along with the name.